src/pred.rs

```diff
@@ -38,13 +38,6 @@ pub fn derive_borders<T: num_traits::AsPrimitive<isize>>(
     _height: usize,
     stride: usize,
 ) -> (isize, isize, isize, isize, isize, isize) {
```

Comment:
shouldn't be changed as well?

Reply:
`plane` is an `isize` so it can assume negative values. Or are we assuming that plane is always an `usize` value?

Reply:
plane is a slice of pixel values, and they are non-negative by definition.

Reply:
That's good, I was deceived by the `isize` in the generic eh eh

Reply:
We use those values to index within u8 so there is lots of room for improvement.